keps/sig-auth/541-external-credential-providers/kep.yaml

```diff
@@ -15,3 +15,11 @@ reviewers:
 approvers:
   - "@liggitt"
   - "@mikedanese"
+prr-approvers:
+  - "@deads2k"
+stage: beta
+latest-milestone: "v1.20"
+milestone:
+  alpha: "v1.10"
+  beta: "v1.11"
+  stable: "v1.21"
```

Comment:
A little confused here, if you're already in beta and going stable in 1.21 - you aren't delivering anything in 1.20? So why the exception?

Reply:
continued beta work, as described in this KEP

Reply:
I'm glad you brought this up since I was a little confused about this too.

I'm not actually sure when we are planning on going stable. I put 1.21 down since I knew we were not going stable in 1.20 (too much to do before freeze), but knew we wanted to do so as soon as we could get the work done. So `stable: "1.21"` might be wrong.

Agreed with @liggitt  on the "continued beta work" comment.

Reply:
I guess i'm wondering are you delivering a betav2 in 1.20 (since old beta was long ago) that you'll want users to know about and need docs etc... or is this just recommencing work to deliver stable in 1.21?

Reply:
I haven't heard of any plans to deliver `v1beta2` in 1.20, so I would assume we are just recommencing work to deliver stable in 1.21. @enj has headed up this effort mostly from sig-auth, and my understanding is that he is pushing us towards a stable milestone, and this feature is just one of the features along the path to that milestone.

> need docs

I do believe we intend to make a doc update for this feature: https://github.com/kubernetes/kubernetes/pull/95489#issue-501604799.